api_v3/lib/types/conversionProfile/filters/KalturaAssetFilter.php

```diff
@@ -20,11 +20,25 @@ public function getMapBetweenObjects()
 	{
 		return array_merge(parent::getMapBetweenObjects(), self::$map_between_objects);
 	}
-	
-	protected function validateEntryIdFiltered()
+
+	/**
+	 * @return array
+	 * @throws KalturaAPIException
+	 */
+	public function validateAndRetrieveEntryIdFiltered()
 	{
-		if(!$this->entryIdEqual && !$this->entryIdIn)
+		if ($this->entryIdEqual)
+		{
+			return array($this->entryIdEqual);
+		}
+		else if ($this->entryIdIn)
+		{
+			return explode(',', $this->entryIdIn);
```

Comment:
Need to test cases where the comma separated EntryIds have spaces in between, in that case do we need to trim or it would work in all flows.

Reply:
this is how we handle all idsIn, we never do handle spaces between comma-separated ids.